benchmarks/problems/WFGtestSuite/wfg.py

```diff
@@ -0,0 +1,706 @@
+from typing import List
+
+import numpy as np
+
+from . import shape_functions
+from . import transformation_functions
```

Comment:
Please use the absolute import.
```suggestion
from benchmarks.problems.wfg import shape_functions
from benchmarks.problems.wfg import transformation_functions
```